### PR TITLE
Add files needed for SiPixel CPE production

### PR DESCRIPTION
**Pull request by tvami**
Add files needed for SiPixel CPE production

### CODE REVIEW

File: tnsnames.ora [hunk 1]
```diff
@@ -0,0 +1,161 @@
+
+
+
+CMS_PXL =
```
Comment (mmusich) [inline tnsnames.ora]: why is this file `tnsnames.ora` needed?

Comment (smuzaffar) [reply tnsnames.ora]: we should not distribute this file. CERN IT does not like it :-(

Comment (tvami) [reply tnsnames.ora]: Thanks, I removed it

**Comment by perrotta** [comment]
+1
- It has to go together with https://github.com/cms-sw/cmssw/pull/39572

**Comment by perrotta** [comment]
merge

**Comment by tvami** [comment]
+alca
 * for the record

**Pull request merged**